tests/test_subquery_is_synced.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import pytest
```

Comment:
```suggestion
import pytest
import os
```
need to add os module in order to get env variables